### DISCUSSION TITLE
History module

### DISCUSSION THREAD

**Discussion by lukaszhandy** [Intermesh/groupoffice#792] [General]
When a user logs in to the history module, System Administrator (ID = 1) is always entered in the user field.
When logging out, the above-mentioned field is completed with the user name (id = user id).
Database field: groupoffice.history_log_entry.createdBy
Is the application working properly?

**Comment by mschering** [comment]
Yes this is normal. When logging it records the active user ID or admin if none is logged in. At the time of the login the user is not logged in yet. Maybe we should allow null instead of the admin user when nobody is logged in.
